app/dashboard/views.py

```diff
@@ -2812,3 +2812,91 @@ def get_hackathons(request):
+    """Per-bounty JSON data for the user dashboard"""
+
+    user = request.user if request.user.is_authenticated else None
+    if not user:
+        return JsonResponse(
+            {'error': _('You must be authenticated via github to use this feature!')},
+            status=401)
+
+    bounty = Bounty.objects.get(id=bounty_id)
+
+    if bounty.status == 'open':
+        interests = Interest.objects.prefetch_related('profile').filter(status='okay', bounty=bounty).all()
+        profiles = [
+            {'interest': {'id': i.id,
+                          'issue_message': i.issue_message},
+             'handle': i.profile.handle,
+             'avatar_url': i.profile.avatar_url,
+             'star_rating': i.profile.get_average_star_rating['overall'],
+             'total_rating': i.profile.get_average_star_rating['total_rating'],
+             'fulfilled_bounties': len(
+                [b for b in i.profile.get_fulfilled_bounties()]),
+             'leaderboard_rank': i.profile.get_contributor_leaderboard_index(),
+             'id': i.profile.id} for i in interests]
+    elif bounty.status == 'submitted':
+        fulfillments = bounty.fulfillments.prefetch_related('profile').all()
+        profiles = []
+        for f in fulfillments:
+            profile = {'fulfiller_metadata': f.fulfiller_metadata}
+            if f.profile:
+                profile.update(
+                    {'handle': f.profile.handle,
+                     'avatar_url': f.profile.avatar_url,
+                     'preferred_payout_address': f.profile.preferred_payout_address,
+                     'id': f.profile.id})
+            profiles.append(profile)
+    else:
+        profiles = []
+
+    return JsonResponse({'title': bounty.title,
+                         'token_name': bounty.token_name,
+                         'value_in_token': bounty.value_in_token,
+                         'value_in_usd': bounty.get_value_in_usdt,
+                         'github_url': bounty.github_url,
+                         'absolute_url': bounty.absolute_url,
+                         'avatar_url': bounty.avatar_url,
+                         'project_type': bounty.project_type,
+                         'expires_date': bounty.expires_date,
+                         'interested_comment': bounty.interested_comment,
+                         'submissions_comment': bounty.submissions_comment,
+                         'profiles': profiles})
+
+
+def funder_dashboard(request):
+    """JSON data for the user dashboard"""
+
+    user = request.user if request.user.is_authenticated else None
+    if not user:
+        return JsonResponse(
+            {'error': _('You must be authenticated via github to use this feature!')},
+            status=401)
+
+    profile = request.user.profile
+
+    open_bounties = list(Bounty.objects.filter(
+        Q(idx_status='open') | Q(idx_status='open'),
+        current_bounty=True,
+        bounty_owner_profile=profile
```

Comment:
Owner profile doesn't get filled a lot of times making the query to don't show nothing

Reply:
Also I have an idx_status open, current_bounty=True, filled bounty_owner_profile=the logged user and the query returns the json without data. Not sure why.

Reply:
Forget that last one, was a problem between the chair and the screen.
But the first one still a problem because sometimes the owner profile doesn't get filled in bounty.

Reply:
Do you think we should fix that the owner profile isn't getting filled in the bounty as expected?